packages/feedback/tests/FeedbackButton.test.js

```diff
@@ -35,4 +35,11 @@ describe('Feedback', () => {
     expect(container.firstChild).toHaveAttribute('aria-pressed', 'false');
     expect(getByText('Home')).toHaveClass('sr-only');
   });
+
+
+  test('should have proper ARIA attributes', () => {
```

Comment:
This test should have a unique name and the lint error on line 41 can be cleaned up